files/mods.json

```diff
@@ -731,6 +731,28 @@
         "forge_id": "skytils",
         "hash": "269236578146f5def7c3f4037efc07e3"
     },
+    {
+        "id": "pss",
```

Comment:
why did you have to call it pss
its just gonna make me think of piss whenever someone says it

Reply:
because its faster than "partly sane skies". also in some places internally (ie the command /pssconfig) it goes by pss

Reply:
okay but you could use that as a nickname
anyway fair its used officially